src/components/pages/dashboard/panels/overview/overviewPanel.js

```diff
@@ -81,7 +81,7 @@ export class OverviewPanel extends Component {
           </div>
         </PanelContent>
         { showOverlay && <PanelOverlay><Indicator /></PanelOverlay> }
-        { error && <PanelError>{ t('errorFormat', { message: t(error.message, { message: error.errorMessage }) }) }</PanelError> }
+        { error && <PanelError><AjaxError t={t} error={error} /></PanelError> }
```

Comment:
Is our general guidelines to have space after `{` ?

Reply:
Yes

Reply:
Not explicitly. I think this is more readable but some auto formatting removes it. Don't have a fix for that atm and it isn't a big enough deal to worry about.